patch-kernel/patch_kernel.sh

```diff
@@ -3,8 +3,8 @@
 set -eu
 DIFF_DIR=$1
 
-if ls ${DIFF_DIR}/*.diff 1>/dev/null 2>&1; then
-  for file in ${DIFF_DIR}/*.diff; do
+if ls "$DIFF_DIR"/*.diff >/dev/null 2>&1; then
+  for file in "$DIFF_DIR"/*.diff; do
```

Comment:
May I ask what is the rationale for removing the curly braces here but not elsewhere? It strikes me as random, to be honest.

Reply:
That's not really needed. I need to remove the `1` which is redundant, and while at it I removed the {}